ios/Beacon/DiceBeaconResponse.m

```diff
@@ -34,15 +36,15 @@ - (id)initWithHTTPURLResponse:(NSHTTPURLResponse *)urlResponse data:(NSData *)da
             if (resp[@"heartbeatInterval"] != nil && [resp[@"heartbeatInterval"] isKindOfClass:NSNumber.class]) {
                 _frequency = ((NSNumber *)resp[@"heartbeatInterval"]).longValue;
             } else {
-                _frequency = 22;
+                _frequency = 5;
             }
         } else {
             // failed to read frequency, failover value set
-            _frequency = 21;
+            _frequency = 5;
         }
         
         if (_frequency < 1) {
-            _frequency = 23;
+            _frequency = 5;
```

Comment:
seems to be some duplication around setting the frequency to 5 in error states. 

```objc
BOOL isValidResponse = error == nil && resp[@"heartbeatInterval"] != nil && [resp[@"heartbeatInterval"] isKindOfClass:NSNumber.class] && [resp[@"heartbeatInterval"] > 1
if (isValidResponse) {
 _frequency = ((NSNumber *)resp[@"heartbeatInterval"]).longValue;
} else _frequency = 5
```

something like that , forgive my poor objc

Reply:
Is was written bit more verbose for readability and ability to set different fallback values, that would help to debug in case we would have any problems.

Reply:
I agree when their were slightly different values set, however in this case seems a bit unnecessary